pkg/querier/stats/stats.proto

```diff
@@ -36,4 +36,7 @@ message Stats {
   google.protobuf.Duration queue_time = 9 [(gogoproto.stdduration) = true, (gogoproto.nullable) = false];
   // The time spent at the frontend encoding the query's final results. Does not include time spent serializing results at the querier.
   google.protobuf.Duration encode_time = 10 [(gogoproto.stdduration) = true, (gogoproto.nullable) = false];
+  // TotalSamples represents the total number of samples scanned while evaluating a query.
+  // This value is taken from the Prometheus engine github.com/prometheus/prometheus/util/stats/query_stats.go
```

Comment:
comment is a bit inaccurate, right? This will also come from MQE. I don't think you should say where it's coming from - that can change with time and outdate the comment.

Reply:
Makes sense, thanks for noticing.